app/core/tests/test_models.py

```diff
@@ -248,3 +251,83 @@ def test_referrer_has_a_referrer_type(referrer):
     assert referrer.referrer_type == bootcamp_referrer_type
     bootcamp_referrer_type.referrer_set.remove(referrer)
     assert bootcamp_referrer_type.referrer_set.count() == 0
+
+
+def test_user_model_old_names():
+    """
+    Test that accessing old field names raises AttributeError.
+    """
+    old_fields = [
+        "current_job_title",
+        "email",
+        "first_name",
+        "gmail",
+        "is_active",
+        "last_name",
+        "preferred_email",
+        "target_job_title",
+    ]
+    for field in old_fields:
+        if not hasattr(User, field):
+            with pytest.raises(
+                AttributeError, match=f"type object 'User' has no attribute '{field}'"
+            ):
+                getattr(User, field)
+
+
+def test_user_has_a_user_status_relationship(user, user2):
+    active_user_status = UserStatusType.objects.get(name="Active")
+    inactive_user_status = UserStatusType.objects.get(name="Inactive")
+    active_user_status.user_set.add(user)
+    active_user_status.user_set.add(user2)
+
+    assert active_user_status.user_set.count() == 2
+
+    assert user.user_status == active_user_status
+    assert user2.user_status == active_user_status
+
+    active_user_status.user_set.remove(user)
```

Comment:
You don't need to remove one status before adding another one. You can just set the user's status to inactive or add the user to the inactive_user_status user_set in the way you're writing it.